Extensions/xkit_patches.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE XKit Patches **//
-//* VERSION 4.0.5 **//
```

Comment:
Probably needs a major version bump per semver, even though we're removing dead code.